### ISSUE TITLE
t/111const-deflate.t tests fail with Compress::Raw::Zlib 2.200

### ISSUE THREAD

**Issue by sdt** [pmqs/IO-Compress#47]
Hello,

This may well be a duplicate of #46 

Compress::Raw::Zlib 2.200 has been released, and the [changes mention zlib_ng](https://metacpan.org/release/PMQS/Compress-Raw-Zlib-2.200/source/Changes#L6)

Installing IO-Compress against that causes test failures in t/111const-deflate.t

```
root@6b6b6136e997:~/.cpanm/work/1655863178.1/IO-Compress-2.106# prove -lv t/111const-deflate.t
t/111const-deflate.t ..
1..355
Failed 355/355 subtests

Test Summary Report
-------------------
t/111const-deflate.t (Wstat: 139 Tests: 0 Failed: 0)
  Non-zero wait status: 139
  Parse errors: Bad plan.  You planned 355 tests but ran 0.
Files=1, Tests=0,  0 wallclock secs ( 0.02 usr  0.00 sys +  0.07 cusr  0.00 csys =  0.09 CPU)
Result: FAIL
```

An easy repro is:

`docker run --rm perl:5.34 cpanm IO::Compress::Base`

Pinning Compress::Raw::Zlib to the previous version "fixes" it:

`docker run --rm perl:5.34 cpanm Compress::Raw::Zlib@2.105 IO::Compress::Base`

**Comment by pmqs** [comment]
Thanks @sdt - issue reproduced

**Comment by pmqs** [comment]
Fixed in 2.201

**Comment by sdt** [comment]
Confirmed working for me with 2.201 - thanks @pmqs !
